plugins/org.eclipse.emf.mwe2.language.ide/src-gen/org/eclipse/emf/mwe2/language/ide/contentassist/antlr/internal/InternalMwe2Parser.tokens

```diff
@@ -0,0 +1,29 @@
+'"'=17
```

Comment:
These token names look suspicious to me

Reply:
?

Reply:
/org.eclipse.xtext.purexbase/src-gen/org/eclipse/xtext/purexbase/parser/antlr/internal/InternalPureXbase.tokens

Reply:
Understood